docs/data/base/components/modal/modal.md

```diff
@@ -0,0 +1,128 @@
+---
+product: base
+title: React Modal component
+components: ModalUnstyled
+githubLabel: 'component: modal'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#dialog_modal
+packageName: '@mui/base'
+---
+
+# Modal
```

Comment:
Some of these pages specify "Unstyled" here and others do not. We should pick one and be consistent. I think it's worthwhile to add "Unstyled."

```suggestion
# Unstyled modal
```